build.sbt

```diff
@@ -107,6 +107,7 @@ lazy val testing = libraryProject("testing")
     description := "Instances and laws for testing http4s code",
     libraryDependencies ++= Seq(
       catsEffectLaws,
+      specs2Cats,
```

Comment:
This is bad, because it pulls in specs2-core.  specs2-matcher is lighter weight and works in other libraries.